doc/ref_cert/README.md

```diff
@@ -45,12 +44,11 @@ deployed the full toolchain in one small virtual machine to verify ONAP Openlab
 via Functest.
 
 <a name="testing-integration-requirements"></a>
-## Test case integration requirements
+## Test Case Integration
 
 To reach all goals (verification, validation, compliance, and conformance)
 expected by CNTT, all test cases must be delivered as
-[Docker containers](https://www.docker.com/) and meet the requirements to
-simplify the CI toolchain setups:
+[Docker containers](https://www.docker.com/) to simplify the CI toolchain setup including:
```

Comment:
I don't understand why we remove requirements. here. All the test case must conform to this rule.
to wrap the last line to 80?

Reply:
It doesn't make sense there in the flow of the sentance

Reply:
Ok.